index.html

```diff
@@ -21,6 +21,21 @@
     <meta name="theme-color" content="#232643" />
     <script type="module" src="/src/main.js"></script>
     <title>Traction</title>
+    <style>
+      /* Inline style for loading pane */
+      #app-loading-pane {
+        width: 20%;
+        margin: 10em auto;
+        text-align: center;
+        font-family: 'Helvetica Neue', Arial, system-ui, -apple-system, BlinkMacSystemFont,
+          'Segoe UI', Roboto, 'Noto Sans', sans-serif;
+        color: #718096;
+      }
+
+      #app-loading-pane-logo {
+        width: 100%;
+      }
+    </style>
```

Comment:
I've added the logo on the loading page. Kept the styles inline so everything loads quickly.